README.md

```diff
@@ -35,18 +35,20 @@ The creator of *OreSpawn*, [**Richard H. Clark (TheyCallMeDanger)**](https://www
 **Therefore, not only we're not going to use the OreSpawn assets, we're also not going to provide, promote or endorse any resource pack that attempts to port them.**
 
 ## Credits/Staff Team
-**Project Lead:** Funky Monk, WeirdNerd
+**Project Leads:** Funky Monk, WeirdNerd
 
-**Animators:** Absolute Kun, Aqu, SnakeMaster
+**Organizers:** Phwee, VDA Spoop
 
-**Builders:** Blackout, Funky Monk, Meme Man, TheLoneCyberman
+**Animators:** Absolute Kun, Aqu, SnakeMaster, Pugman665544
```

Comment:
All of these should be in alphabetical order and all the lines below it. With T4 going last since it isn't a normal font.